vuepress/docs/.vuepress/components/LandingPage.vue

```diff
@@ -51,32 +49,31 @@
-            <div class="language-sh"><pre>IP=MY_OPENSHIFT_IP.nip.io</pre></div>
-            <p>Update the Helm chart using your IP</p>
-            <div class="language-sh"><pre>sed -i "s/192.168.64.25.nip.io/$IP/" entando-okd4.yaml</pre></div>
+            <p>Install the namespace-scoped Entando custom resources</p>
+            <div class="language-sh"><pre>oc apply -n entando -f https://raw.githubusercontent.com/entando/entando-releases/v7.0.1/dist/ge-1-1-6/namespace-scoped-deployment/namespace-resources.yaml</pre></div>
+            <p>Download the `entando-app.yaml` template</p>
+            <div class="language-sh"><pre>curl -sLO "https://raw.githubusercontent.com/entando/entando-releases/v7.0.1/dist/ge-1-1-6/samples/entando-app.yaml"</pre></div>
+            <p>Edit `entando-app.yaml` with the hostname for your application</p>
+            <div class="language-sh"><pre>e.g. YOUR-HOST-NAME = quickstart.YOUR-IP.nip.io</pre></div>
           </div>
 
           <hr class="get-started-separator" />
 
           <h3 id="step-3" @click="toggleStepThree($event)">Deploy Entando</h3>
           <div style="display:none">
-            <p>Create Kubernetes objects to define your cluster's desired state</p>
-            <div class="language-sh"><pre>oc create -f entando-okd4.yaml</pre></div>
-            <p>Watch the installation until the cluster is ready for use, indicated by a pod named <span>quickstart-server-*</span> with 3/3 in the READY column and RUNNING in the STATUS column. Use CTRL-C to stop watching the deployment</p>
+            <p>Install Entando</p>
+            <div class="language-sh"><pre>oc apply -n entando -f entando-app.yaml</pre></div>
+            <p>Watch the pods warming up. Use CTRL+C to stop watching the deployment.</p>
             <div class="language-sh"><pre>oc get pods -n entando --watch</pre></div>
-            <p>Get the URL to access Entando from your local browser, e.g. <span>quickstart-entando.192.168.64.25.nip.io/app-builder/</span></p>
-            <div class="language-sh"><pre>oc get ingress -n entando -o jsonpath='{.items[2].spec.rules[*].host}{.items[2].spec.rules[*].http.paths[2].path}{"\n"}'</pre></div>
-            <p>Login to the <span>Entando App Builder</span> with username:<span>admin</span>, password: <span>adminadmin</span></p>
-            <p>Choose an action from the <a :href="path('/docs/getting-started/#next-steps')">Next Steps</a> list to continue your journey with Entando!</p>
+            <p>Access Entando from your local browser</p>
+            <div class="language-sh"><pre>http://YOUR-HOST-NAME/app-builder/, e.g. quickstart.YOUR-IP.nip.io/app-builder/</pre></div>
+            <p>Login to the Entando App Builder</p>
+            <div class="language-sh"><pre>username: admin, password: adminadmin</pre></div>
+            <p>Choose an action from <a :href="path('/docs/getting-started/#next-steps')">Next Steps</a> to continue your journey with Entando!</p>
```

Comment:
does it sound awkward without the or something >> "Choose an action from the Next Steps to continue...."

Reply:
i actually think we should leave out "the"; Next Steps without an article prefacing it is more direct, and there are many incarnations of "next steps," so "the" Next Steps implying that there's only one set of next steps is misleading...